recipes/abacus/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+mpi:
+  - mpich
+  - openmpi
```

Comment:
Maybe keeping MPICH only is enough?

```suggestion
```

Reply:
Conda-forge suggests supporting both MPIs. See https://conda-forge.org/docs/maintainer/knowledge_base.html#mpi-variants-in-conda-forge